test/expect-tests/process_tests.ml

```diff
@@ -18,17 +18,15 @@ let true_ =
 
 let%expect_test "null input" =
   let stdin_from = Process.(Io.null In) in
-  let run () =
-    Process.run ~display:!Clflags.display ~stdin_from Strict true_ []
-  in
+  let run () = Process.run ~display:Quiet ~stdin_from Strict true_ [] in
```

Comment:
In this test, we globally set the display to quiet. Here we are being more explicit. When we run these processes we don't want/need to see it reported to the user. It is also just a test.